lib/ash_json_api/json_schema/open_api.ex

```diff
@@ -383,16 +383,27 @@ if Code.ensure_loaded?(OpenApiSpex) do
     end
 
     def tags(api) do
-      api
-      |> resources()
-      |> Enum.map(fn resource ->
-        name = AshJsonApi.Resource.Info.type(resource)
+      open_api = AshJsonApi.Api.Info.open_api(api)
 
-        %Tag{
-          name: to_string(name),
-          description: "Operations on the #{name} resource."
-        }
-      end)
+      if open_api && open_api[:group_by] == :api && open_api[:tag] do
```

Comment:
We should probably raise an error at compile time if they don't include a tag but they do include a `group_by :api`.

Reply:
This would be done with a `Spark.Dsl.Verifier`. There should be other examples of verifiers.